test/models/friendship_test.rb

```diff
@@ -0,0 +1,38 @@
+require 'test_helper'
+
+class FriendshipTest < ActiveSupport::TestCase
+  def setup
+    @user = users(:musa)
+    @friend = users(:kingsley)
+    @friendship = Friendship.new(user_id: @user.id, friend_id: @friend.id)
+  end
+
+  test 'should be valid' do
```

Comment:
We can safely remove the tests for the presence of friend_id, user_id as the association tests themselves take care of validating the presence of these two attributes in a friendship row.